apps/frontend-manage/src/components/courses/PointCorrectionsModal.tsx

```
@@ -0,0 +1,433 @@
+    const scopeSchema = Yup.object({
+      scopeType: Yup.mixed<CorrectionScope>()
+        .oneOf(['instance', 'quiz'])
+        .required(),
+      quizId: Yup.string()
+        .trim()
+        .required(t('manage.pointCorrections.errorQuizRequired')),
+      instanceId: Yup.string()
+        .trim()
+        .when('scopeType', {
+          is: 'instance',
+          then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)